### PR TITLE
Enhance the `Jalali BitDatePicker` sample in the `BitDatePicker` demo page (#2420)

### LINKED ISSUE #2420: Enhance the `Jalali BitDatePicker` code sample in the `Playground` project
The code sample of the `Jalali BitDatePicker` which you can see [here](https://components.bitplatform.dev/components/date-picker#example3) should be completed based on the last enhancements.

### PR DESCRIPTION
**Pull request by msangtarash**
this closes #2420

### CODE REVIEW

File: src/BlazorUI/Bit.BlazorUI.Tests/Pickers/BitDatePickerTests.cs [hunk 1]
```diff
@@ -114,12 +114,12 @@ public void BitDatePickerCalendarSelectTodayDate()
     public void BitDatePickerCalendarWithCustomCultureInfo()
     {
         Context.JSInterop.Mode = JSRuntimeMode.Loose;
-        var component = RenderComponent<BitDatePickerTest>(
-           parameters =>
+        var component = RenderComponent(
+           (Action<ComponentParameterCollectionBuilder<BitDatePickerTest>>)(           parameters =>
```
Comment (ysmoradi) [inline src/BlazorUI/Bit.BlazorUI.Tests/Pickers/BitDatePickerTests.cs]: Revet this

**Review by ysmoradi — APPROVED** [review]

**Review by msynk — CHANGES_REQUESTED** [review]

File: src/BlazorUI/Bit.BlazorUI/Utils/CultureInfoHelper.cs [hunk 2]
```diff
@@ -1,10 +1,58 @@
...
+        var cultureInfo = CultureInfo.CreateSpecificCulture("fa-IR");
+
+        cultureInfo.DateTimeFormat.MonthNames = new[]
+        {
+            "فروردین", "اردیبهشت", "خرداد", "تیر", "مرداد", "شهریور", "مهر", "آبان", "آذر", "دی", "بهمن", "اسفند", ""
+        };
+
+        cultureInfo.DateTimeFormat.AbbreviatedMonthNames = new[]
+        {
+            "فروردین", "اردیبهشت", "خرداد", "تیر", "مرداد", "شهریور", "مهر", "آبان", "آذر", "دی", "بهمن", "اسفند", ""
```
[the remark this answers is not in the document]
Comment (msangtarash) [reply src/BlazorUI/Bit.BlazorUI/Utils/CultureInfoHelper.cs:23]: I have no idea.
The only value I can offer is using numbers
1 2 3 ...12 :)

[the remark this answers is not in the document]
Comment (msynk) [reply src/BlazorUI/Bit.BlazorUI/Utils/CultureInfoHelper.cs:23]: hmm, what about something like these:
"فرور", "ارد", "خرد", "تیر", "مرد", "شهر", "مهر", "آبا", "آذر", "دی", "بهم", "اسف"

File: src/BlazorUI/Bit.BlazorUI/Utils/CultureInfoHelper.cs [hunk 3]
```diff
@@ -44,19 +92,7 @@ public static CultureInfo GetPersianCultureByFinglishNames()
...
-            "YekShanbe",
-            "DoShanbe",
-            "SeShanbe",
-            "ChaharShanbe",
-            "PanjShanbe",
-            "Jome",
-            "Shanbe"
-        };
-
-        cultureInfo.DateTimeFormat.AbbreviatedDayNames = new[]
```
[the remark this answers is not in the document]
Comment (ysmoradi) [reply src/BlazorUI/Bit.BlazorUI/Utils/CultureInfoHelper.cs:59]: ?

File: src/BlazorUI/Bit.BlazorUI/Utils/CultureInfoHelper.cs [hunk 4]
```diff
@@ -78,6 +114,26 @@ public static CultureInfo GetPersianCultureByFinglishNames()
...
+            "D",
+            "S",
+            "C",
+            "P",
+            "J",
+            "S"
+        };
+
+        cultureInfo.DateTimeFormat.AMDesignator = "ق.ظ";
+        cultureInfo.DateTimeFormat.PMDesignator = "ب.ظ";
```
[the remark this answers is not in the document]
Comment (ysmoradi) [reply src/BlazorUI/Bit.BlazorUI/Utils/CultureInfoHelper.cs:129]: Do you mean GH.Z & B.Z?

File: src/BlazorUI/Playground/Bit.BlazorUI.Playground/Web/Pages/Components/DatePicker/BitDatePickerDemo.razor [hunk 5]
```diff
@@ -26,7 +26,7 @@
         </ExamplePreview>
     </ComponentExampleBox>
 
-    <ComponentExampleBox Title="Jalali BitDatePicker with Finglish names" HTMLSourceCode="@example4HTMLCode" CSharpSourceCode="@example4CSharpCode" ExampleId="example4">
+    <ComponentExampleBox Title="Jalali BitDatePicker with Finglish names" HTMLSourceCode="@example4HTMLCode" ExampleId="example4">
```
[the remark this answers is not in the document]
Comment (ysmoradi) [reply src/BlazorUI/Playground/Bit.BlazorUI.Playground/Web/Pages/Components/DatePicker/BitDatePickerDemo.razor:29]: There's no need to CultureInfoHelper code anymore

**Review by msynk — APPROVED** [review]

**Pull request merged**